pkg/storage/ingest/reader.go

```diff
@@ -898,6 +922,10 @@ type concurrentFetchers struct {
 	trackCompressedBytes bool
 }
 
+func (r *concurrentFetchers) Stop() {
+	r.stop()
```

Comment:
Because we also use `stop` in other functions, I'd rather keep it private the distinction is subtle but there's a high chance we might want to do other stuff as part of stopping in the future and it clearly signals there there's both a private and a public method.

Reply:
in the interest of merigng this faster, ok, but having two methods just in case sounds weird. it's cheap to add a method later on